presto-native-execution/presto_cpp/main/operators/tests/PlanNodeSerdeTest.cpp

```diff
@@ -71,7 +71,7 @@ TEST_F(PlanNodeSerdeTest, partitionAndSerializeNode) {
                   .values(data_, true)
                   .addNode(addPartitionAndSerializeNode(
                       4, false, reverseColumns(asRowType(data_[0]->type()))))
-                  .localPartition({})
+                  .localPartition(std::vector<std::string>{})
```

Comment:
Are these changes needed?

Reply:
Yes, there is ambiguity if we just {} as localPartition could take a shared ptr of BucketedFunctionProperty as input